baselines/ippo/config/ippo_ff_puffer.yaml

```diff
@@ -2,36 +2,60 @@ mode: "train"
 use_rnn: false
 eval_model_path: null
 baseline: false
-data_dir: "data/processed/examples" # Dataset
+data_dir: "data/processed/validation"
 
 environment: # Overrides default environment configs (see pygpudrive/env/config.py)
   name: "gpudrive"
-  num_worlds: 50 # Number of parallel environments
-  k_unique_scenes: 1 # Number of unique scenes to sample from
+  num_worlds: 100 # Number of parallel environments
+  k_unique_scenes: 100 # Number of unique scenes to sample from
   max_controlled_agents: 32 # Maximum number of agents controlled by the model. Make sure this aligns with the variable kMaxAgentCount in src/consts.hpp
   ego_state: true
   road_map_obs: true
   partner_obs: true
   normalize_obs: true
-  reward_type: "sparse_on_goal_achieved"
+  remove_non_vehicles: true # If false, all agents are included (vehicles, pedestrians, cyclists)
+  use_lidar_obs: false # NOTE: Setting this to true currently turns of the other observation types
+  reward_type: "weighted_combination"
+  collision_weight: -0.025
+  off_road_weight: -0.025
+  goal_achieved_weight: 1.0
   dynamics_model: "classic"
-
+  collision_behavior: "ignore" # Options: "remove", "stop"
+  dist_to_goal_threshold: 3.0
+  polyline_reduction_threshold: 0.2 # Rate at which to sample points from the polyline (0 is use all closest points, 1 maximum sparsity), needs to be balanced with kMaxAgentMapObservationsCount
+  sampling_seed: 42 # If given, the set of scenes to sample from will be deterministic, if None, the set of scenes will be random
+  obs_radius: 60.0 # Visibility radius of the agents
 wandb:
   entity: ""
   project: "gpudrive"
-  group: "my_group"
+  group: "talk_figures"
   mode: "online" # Options: online, offline, disabled
-  tags: ["ppo", "ff", "single_scene"]
+  tags: ["ppo", "ff"]
+
+## NOTES
+## Good batch size: 128 * number of controlled agents (e.g. 2**18)
+## Minibatch size 1/16 of batch size, eg. 16_000
 
 train:
-  exp_id: null  # Set dynamically in the script if needed
+  exp_id: PPO # Set dynamically in the script if needed
   seed: 42
   cpu_offload: false
   device: "cuda"  # Dynamically set to cuda if available, else cpu
+  bptt_horizon: 2
+  compile: false
+  compile_mode: "reduce-overhead"
+
+  # # # Data sampling # # #
+  resample_scenes: false
+  resample_criterion: "global_step"
+  resample_interval: 5_000_000
+  resample_mode: "random" # Options: random
+
+  # # # PPO # # #
   torch_deterministic: false
-  total_timesteps: 2_000_0000
-  batch_size: 25_000
-  minibatch_size: 5_000
+  total_timesteps: 500_000_000
+  batch_size: 65_536
+  minibatch_size: 16_384
```

Comment:
What is the difference between these two btw?

Reply:
Batch size is the size of the rollout buffer, mini batch size is the size of the mini batch used for updating the policy

Reply:
Isn't this a really small rollout buffer size? If you have 100 world with 10 agents I guess this is roughly equivalent to training every 65 sim steps?

Reply:
We don't have to change it in this PR, just discussing it here.

Reply:
Good point, I think I should set it to at least 91 sim steps in the future.